experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -353,7 +353,7 @@ dates = (; date = [date], date0 = [date0], date1 = [Dates.firstdayofmonth(date0)
 User can write custom diagnostics in the `user_diagnostics.jl`.
 =#
 monthly_3d_diags = init_diagnostics(
-    (:T, :u, :q_tot),
+    (:T, :u, :q_tot, :q_liq_ice),
```

Comment:
Variable changes : `q_liq_ice` (addition)  ,`toa_fluxes` (renaming), `turbulent_energy_fluxes` (addition) . each then requires an `operations` spec, which exists, plus use in a call to the `paperplots` function, which is also OK. Overall structure LGTM.